### ISSUE TITLE
Cannot install on Macbook M1

### ISSUE THREAD

**Issue by ispmarin** [erdewit/HiFiScan#4]
I get the following error:

```bash
pip install hifiscan
Collecting hifiscan
  Using cached hifiscan-1.0.1-py3-none-any.whl (14 kB)
Collecting numpy
  Using cached numpy-1.23.3-cp310-cp310-macosx_11_0_arm64.whl (13.3 MB)
Collecting sounddevice
  Using cached sounddevice-0.4.5-py3-none-macosx_10_6_x86_64.macosx_10_6_universal2.whl (108 kB)
Collecting eventkit
  Downloading eventkit-0.8.9-py3-none-any.whl (31 kB)
Collecting pyqtgraph
  Using cached pyqtgraph-0.12.4-py3-none-any.whl (995 kB)
Collecting numba
  Using cached numba-0.56.2-cp310-cp310-macosx_11_0_arm64.whl (2.5 MB)
Collecting PyQt5
  Using cached PyQt5-5.15.7.tar.gz (3.2 MB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Preparing metadata (pyproject.toml) ... error
  error: subprocess-exited-with-error

  × Preparing metadata (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> [29 lines of output]
      Traceback (most recent call last):
        File "/Users/imarin/.local/lib/venv/hifi2/lib/python3.10/site-packages/pip/_vendor/pep517/in_process/_in_process.py", line 156, in prepare_metadata_for_build_wheel
          hook = backend.prepare_metadata_for_build_wheel
      AttributeError: module 'sipbuild.api' has no attribute 'prepare_metadata_for_build_wheel'

      During handling of the above exception, another exception occurred:

      Traceback (most recent call last):
        File "/Users/imarin/.local/lib/venv/hifi2/lib/python3.10/site-packages/pip/_vendor/pep517/in_process/_in_process.py", line 363, in <module>
          main()
        File "/Users/imarin/.local/lib/venv/hifi2/lib/python3.10/site-packages/pip/_vendor/pep517/in_process/_in_process.py", line 345, in main
          json_out['return_val'] = hook(**hook_input['kwargs'])
        File "/Users/imarin/.local/lib/venv/hifi2/lib/python3.10/site-packages/pip/_vendor/pep517/in_process/_in_process.py", line 160, in prepare_metadata_for_build_wheel
          whl_basename = backend.build_wheel(metadata_directory, config_settings)
        File "/private/var/folders/dt/wtzqf0dx24bc2x45fgsfhk480000gn/T/pip-build-env-81nvefxk/overlay/lib/python3.10/site-packages/sipbuild/api.py", line 46, in build_wheel
          project = AbstractProject.bootstrap('wheel',
        File "/private/var/folders/dt/wtzqf0dx24bc2x45fgsfhk480000gn/T/pip-build-env-81nvefxk/overlay/lib/python3.10/site-packages/sipbuild/abstract_project.py", line 87, in bootstrap
          project.setup(pyproject, tool, tool_description)
        File "/private/var/folders/dt/wtzqf0dx24bc2x45fgsfhk480000gn/T/pip-build-env-81nvefxk/overlay/lib/python3.10/site-packages/sipbuild/project.py", line 584, in setup
          self.apply_user_defaults(tool)
        File "/private/var/folders/dt/wtzqf0dx24bc2x45fgsfhk480000gn/T/pip-install-knjn99ng/pyqt5_bb08a737cc0444979dff73d867746dba/project.py", line 69, in apply_user_defaults
          super().apply_user_defaults(tool)
        File "/private/var/folders/dt/wtzqf0dx24bc2x45fgsfhk480000gn/T/pip-build-env-81nvefxk/overlay/lib/python3.10/site-packages/pyqtbuild/project.py", line 70, in apply_user_defaults
          super().apply_user_defaults(tool)
        File "/private/var/folders/dt/wtzqf0dx24bc2x45fgsfhk480000gn/T/pip-build-env-81nvefxk/overlay/lib/python3.10/site-packages/sipbuild/project.py", line 236, in apply_user_defaults
          self.builder.apply_user_defaults(tool)
        File "/private/var/folders/dt/wtzqf0dx24bc2x45fgsfhk480000gn/T/pip-build-env-81nvefxk/overlay/lib/python3.10/site-packages/pyqtbuild/builder.py", line 67, in apply_user_defaults
          raise PyProjectOptionException('qmake',
      sipbuild.pyproject.PyProjectOptionException
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.
```

**Comment by erdewit** [comment]
It seems PyQt5 has no Apple arm64 support. I'll look into switching to PyQt6 or PySide6

**Comment by erdewit** [comment]
The just released 1.0.2 version uses PySide6, for which there is an Apple universal wheel.
